content/06.methods.md

```diff
@@ -290,12 +290,11 @@ The fusion filtering workflow can be found in the [OpenPBTA Analysis repository]
 #### Oncoprint figure generation
 
 We used Maftools [@doi:10.1101/gr.239244.118] to generate oncoprints depicting the frequencies of canonical somatic gene mutations, CNVs, and fusions for the top 20 genes mutated across primary tumors within broad histologies of the OpenPBTA dataset.
-We collated canonical genes from the literature for low-grade astrocytic tumors [@doi:10.1186/s40478-020-00902-z], embryonal tumors [@doi:10.1038/nature22973; @doi:10.1007/s00401-020-02182-2; @doi:10.1186/s40478-020-00984-9; @doi:10.1016/j.ccell.2016.02.001; @doi:10.1038/s41598-020-59812-8], diffuse astrocytic and oligodendroglial tumors [@doi:10.1016/j.ccell.2017.08.017; @doi:10.1002/ijc.32258; @doi:10.1093/neuonc/noab106; @doi:10.1186/s40478-020-00905-w],  and other tumors (ependymal tumors, craniopharyngiomas, neuronal-glial mixed tumors, histiocytic tumors, chordoma, meningioma, and choroid plexus tumors) [@doi:10.1186/s40478-020-01056-8; @doi:10.1007/s00381-017-3481-3; @doi:10.1016/j.ccell.2015.04.002;]
-@doi:10.1038/nature13109; @doi:10.1038/s41525-017-0014-7; @doi:10.3171/2019.8.JNS191266; @doi:10.1007/s00401-016-1539-z; @doi:10.1093/neuonc/noaa267; @doi:10.1016/s0002-9440(10)64477-x; @doi:10.1016/j.jaad.2017.05.059].
+We collated canonical genes from the literature for low-grade astrocytic tumors [@doi:10.1186/s40478-020-00902-z], embryonal tumors [@doi:10.1038/nature22973; @doi:10.1007/s00401-020-02182-2; @doi:10.1186/s40478-020-00984-9; @doi:10.1016/j.ccell.2016.02.001; @doi:10.1038/s41598-020-59812-8], diffuse astrocytic and oligodendroglial tumors [@doi:10.1016/j.ccell.2017.08.017; @doi:10.1002/ijc.32258; @doi:10.1093/neuonc/noab106; @doi:10.1186/s40478-020-00905-w],  and other tumors (ependymal tumors, craniopharyngiomas, neuronal-glial mixed tumors, histiocytic tumors, chordoma, meningioma, and choroid plexus tumors) [@doi:10.1186/s40478-020-01056-8; @doi:10.1007/s00381-017-3481-3; @doi:10.1016/j.ccell.2015.04.002; @doi:10.1038/nature13109; @doi:10.1038/s41525-017-0014-7; @doi:10.3171/2019.8.JNS191266; @doi:10.1007/s00401-016-1539-z; @doi:10.1093/neuonc/noaa267; @doi:10.1016/s0002-9440(10)64477-x; @doi:10.1016/j.jaad.2017.05.059].
```

Comment:
Something's up with the very last set of references in the HTML version from AppVeyor _but_ not sure what it could be given that the first one that renders correctly is DOI: 10.1186/s40478-020-01056-8.

Reply:
hmm, weird. oh i wonder if for some reason it thinks it should be a link with other tumors in parentheses before brackets.. let me remove.

Reply:
nope, that didn't fix it...

Reply:
ok @jaclyn-taroni - I do not know what else to do here... I tried PMID, tried adding the first ref to the end, but none of that is working - it is bizarre.